src/Javis.jl

```diff
@@ -69,6 +82,37 @@ mutable struct Action <: ActionType
     opts                    :: Dict{Symbol, Any}
 end
 
+"""
+    Rel
+
+Ability to define frames in a relative fashion.
+
```

Comment:
Similar to what you did for the docstring for `Video` could you also define the fields of Rel?